Sparkle/SUUpdater.m

```diff
@@ -303,7 +303,16 @@ - (void)checkForUpdatesInBackground
     // Background update checks should only happen if we have a network connection.
     //	Wouldn't want to annoy users on dial-up by establishing a connection every
     //	hour or so:
-    SUUpdateDriver *theUpdateDriver = [[([self automaticallyDownloadsUpdates] ? [SUAutomaticUpdateDriver class] : [SUScheduledUpdateDriver class])alloc] initWithUpdater:self];
+    SUUpdateDriver *theUpdateDriver;
```

Comment:
Might be good to set this to nil (as we check for it later).

I know Apple says [here] (https://developer.apple.com/library/ios/releasenotes/ObjectiveC/RN-TransitioningToARC/Introduction/Introduction.html#//apple_ref/doc/uid/TP40011226-CH1-SW5) that it explicitly sets autoreleasing stack variables to nil, but we know how Apple can flip-flop :-)

I wouldn't worry about this change unless you need to do another build.